testing/src/main/java/org/oppia/android/testing/media/TestMediaPlayer.kt

```diff
@@ -0,0 +1,51 @@
+package org.oppia.android.testing.media
+
+import android.content.Context
+import android.media.MediaPlayer
+
+/**
+ * Helper class to test Media Player related test cases in both Robolectric & Espresso.
+ */
+interface TestMediaPlayer {
+
+  /**
+   * Add Media details and Audio Data Source
+   *
+   * Adding the required details of media which we want our test cases to run on.
+   * [context] the test environment context
+   * [explorationId1], [explorationId2] Id of the exploration
+   * [audioFileName1], [audioFileName2] name of the  audio file
+   * [mediaPlayer] Media Player object we use in our test case
+   */
+  fun addMediaInfo(
+    context: Context,
+    explorationId1: String,
+    explorationId2: String,
+    audioFileName1: String,
+    audioFileName2: String,
```

Comment:
For here & below, why not just call the function multiple times for subsequent audio sources? Could we design the API to do that?